src/styles/Fonts.ts

```diff
@@ -29,22 +29,22 @@ export const FontsRaw = {
 
   HeadingXS: {
     ...sharedFontStyles,
-    lineHeight: '28px',
+    lineHeight: '1.5',
     fontSize: '24px',
     fontFamily: 'Metropolis',
   },
 
   BodyDefault: {
     ...sharedFontStyles,
-    lineHeight: '16px',
+    lineHeight: '1.5',
```

Comment:
Please update the `body` in `components.js`. Look for `MuiTypography`, `styleOverrides` for `body1`.

Also, could you please remove all references of `BodySmall` and `BodySmallDark` in in this file? They are not being used anywhere in the project.

Reply:
`HeadingXS` and `HeadingXSDark` were being used in the project so I left them as they were (used in `ResultsMain.tsx` and `SearchContainerStyles.ts`) and removed `BodySmall` and `BodySmallDark` from Fonts and their references as they were not actually being used in the project.